src/server.cpp

```diff
@@ -705,7 +707,12 @@ void CServer::SetEnableRecording ( bool bNewEnableRecording )
 {
     if ( bRecorderInitialised )
     {
+        // note that this block executes regardless of whether
+        // what appears to be a change is being applied, to ensure
+        // the requested state is the result
+
         bEnableRecording = bNewEnableRecording;
+        qInfo() << "Recording state" << ( bEnableRecording ? "enabled" : "disabled" );
```

Comment:
You need to check the Qt version to use qInfo: https://doc.qt.io/qt-5/qtglobal.html#qInfo

Reply:
See, e.g., here: https://github.com/corrados/jamulus/blob/master/src/clientdlg.cpp#L763

Reply:
Ah right, will do.